src/main/java/com/vmware/gemfire/testcontainers/GemFireCluster.java

```diff
@@ -583,7 +587,12 @@ public String gfsh(boolean logOutput, String... commands) {
     if (commands[0].startsWith("connect")) {
       fullCommand = "";
     } else {
-      fullCommand = String.format("connect --jmx-manager=localhost[%d]\n", JMX_PORT);
+      fullCommand = String.format("connect --jmx-manager=localhost[%d]", JMX_PORT);
+      if (credentials != Credentials.NONE) {
+        fullCommand += " --username=" + credentials.getUsername();
+        fullCommand += " --password=" + credentials.getPassword();
```

Comment:
How would all of this work if username/password as set in the `--security-properties-file`?